packages/design-system-react/package.json

```diff
@@ -76,7 +76,7 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/design-tokens": "^4.1.0",
+    "@metamask/design-tokens": "^5.0.0",
```

Comment:
We're bumping to 5.0.0 because we're making an intentional breaking change to fix type imports (which were unintentionally broken in 4.2.0 during the monorepo migration). This requires users to update their import paths from deep imports to root imports, and necessitates updating peer dependencies in related packages